src/features/slackapi/slackapi-utils.ts

```diff
@@ -169,11 +169,24 @@ Have a great week!
 
       const message = constructDailyMessage(userData, numberOfToday);
 
+      const listOfIds = process.env.STAGING_IDS.split(",");
+
+
       if (!isAway && !firstDayBack) {
-        messageResults.push({
-          message: message,
-          response: await sendMessage(slackId, message.message)
-        });
+        if (listOfIds.length === 0) {
```

Comment:
A cleaner conditional usage could be if (!listOfIds), but of course check this still works as intended